### PR TITLE
allow usage of ltr

### PR DESCRIPTION
**Pull request by Jibec**
since v0.67.1 it is possible to define language direction
see https://github.com/gohugoio/hugo/issues/6550

### CODE REVIEW

**Comment by Jibec** [comment]
well... I just discovered this works fine:
```
[languages.ar.params.settings]
rtl = true
```

is it really useful to continue this work?

File: layouts/partials/head/includes.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <link rel='icon' href='{{ ( or .Site.Params.assets.favicon "favicon.ico" ) | relURL }}'>
 <link rel='stylesheet' href='{{ print "/assets/css/" .Site.Data.assets.main.css | relURL  }}'>
 
-{{- if .Site.Params.settings.rtl -}}
+{{- if eq .Site.Language.LanguageDirection "rtl" -}}
```
Comment (MunifTanjim) [inline layouts/partials/head/includes.html:4]: Please, don't remove the check for existing option, that can potentially break existing sites using that option.

```suggestion
{{- if or ( .Site.Params.settings.rtl ) ( eq .Site.Language.LanguageDirection "rtl" ) -}}
```

Let's keep it for backward compatibility.

**Comment by MunifTanjim** [comment]
> is it really useful to continue this work?

As Hugo now supports setting this option officially, using that will be the right thing to do. :smiley:

**Comment by Jibec** [comment]
> > is it really useful to continue this work?
> 
> As Hugo now supports setting this option officially, using that will be the right thing to do. smiley

yeah, I figured it out after a minute ;)

**Comment by MunifTanjim** [comment]
@Jibec Thanks so much :smiley: :tada:

**Comment by Jibec** [comment]
you are welcome, I'm going to continue as you provide great guidance

**Pull request merged**